src/models/parallelCoordinates.js

```diff
@@ -9,16 +9,17 @@ nv.models.parallelCoordinates = function() {
     // Public Variables with Default Settings
     //------------------------------------------------------------
 
-    var margin = {top: 30, right: 10, bottom: 10, left: 10}
-        , width = null
-        , height = null
+    var margin = {top: 30, right: 0, bottom: 10, left: 0}
+        , width = 1000
+        , height = 500
```

Comment:
These should start off being null so it uses the full width and height of the SVG, if you need specific values the chart.width() and chart.height() options should be used.